### PR TITLE
NGSIv1 removal 

### PR DESCRIPTION
**Pull request by fgalan**
This PR is not for being merged, it is just to expose publicly the branch. PR will be closed (and the branch deleted) after CI e2e testing.

### CODE REVIEW

**Comment by fgalan** [comment]
After merging PRs #487 #499 #505 and upgrade this PR branch with master, tests pass to ok. 

However, not sure if the NGSI-v1 removal work has ended in this repo... we have recently discovered `isCurrentNgsi()` usages in IOTA code. We are investigating it.

**Comment by fgalan** [comment]
Usage of isCurrentNgsi():

https://github.com/telefonicaid/iotagent-ul/search?q=isCurrentNgsi

**Comment by mapedraza** [comment]
Changes done in https://github.com/telefonicaid/iotagent-json/pull/533 replicated here

**Review by fgalan — COMMENTED** [review]
LGTM to code changes, although not ready to be merged yet (first packages.json should be changed back).

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Fix: check access to active attribute of device before use it (twin of iota-json#576)
 Fix: processing configurations subscriptions in NGSIv2 (twin of iota-json#563)
+Remove: NGSI-v1 specific behaviours (twin of iota-ul#533)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:3]:
```suggestion
Remove: NGSI-v1 specific behaviours (twin of iota-json#533)
```

Comment (fgalan) [reply CHANGES_NEXT_RELEASE]: Fixed in d0df1d5

**Comment by fgalan** [comment]
> LGTM to code changes, although not ready to be merged yet (first packages.json should be changed back).

Done in 52ebdeb

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**